app/src/common/shared/com/igalia/wolvic/utils/EnvironmentsManager.java

```diff
@@ -146,12 +157,15 @@ public void onDownloadsUpdate(@NonNull List<Download> downloads) {
 
     @Override
     public void onDownloadCompleted(@NonNull Download download) {
+        assert download.getStatus() == Download.SUCCESSFUL;
+        if (download.getOutputFile() == null) {
```

Comment:
I guess with the new changes above this should really be an assert? (I mean because we don't call it on error)

Reply:
> I guess with the new changes above this should really be an assert? (I mean because we don't call it on error)

Yeah, but I thought it was useful lo have a log with the missing input stream's URI and avoid the crash if that scenario happens.